lib/create-contentful-api.js

```diff
@@ -41,7 +41,7 @@
 import { createRequestConfig } from 'contentful-sdk-core'
 import errorHandler from './error-handler'
 import entities from './entities'
-
+import {wrapUser} from './entities/user'
```

Comment:
so this was the solution to the import problem? :(

I have a bad feeling with having the entities imported twice, one general object with all entities as property plus wrapUser from the user separate? 👎

Reply:
Should be at the beginning of `createClientApi` as `wrapSpace` and `wrapOrganizationCollection`.

I can give it a try? 🤷‍♂️

Reply:
I won't merge the PR yet until I find out what is going on with the entities